src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java.mdx

```diff
@@ -865,7 +865,10 @@ Once you've reached the `getting-started-java` entity in New Relic:
       title="Screenshot showing the main distributed tracing page"
       src={opentelemetryMainDistributedTracingPage}
     />
-2. From there, find a trace with an error and click to open it:
+2. From there, find a trace with an error and click to open it. 
+    <Callout variant="tip">
+      With the OpenTelemetry Java agent, you might see different result than you'd see with the New Relic APM Java agent. This is because there's a difference in error handling between the two agent types. For more information, review [Handling error rate in OpenTelemetry and New Relic](https://newrelic.com/blog/how-to-relic/error-rate-opentelemetry-and-new-relic).
```

Comment:
```suggestion
      With the OpenTelemetry Java agent, you might see different result than you'd see with the New Relic APM Java agent. This is because there's a difference in error handling between the two agent types. For more information, see [Handling error rate in OpenTelemetry and New Relic](https://newrelic.com/blog/how-to-relic/error-rate-opentelemetry-and-new-relic).
```